tests/views/admin.py

```diff
@@ -9,12 +9,83 @@
 
 import pytest
 
-from django.core.urlresolvers import reverse_lazy
+from django.core.urlresolvers import reverse_lazy, reverse
+from django.utils.safestring import mark_safe
+
+from pytest_pootle.env import TEST_USERS
+from pytest_pootle.factories import LanguageFactory
+
+from pootle.core.paginator import paginate
+from pootle.core.url_helpers import split_pootle_path
+from pootle_app.views.admin.util import form_set_as_table
+from pootle_project.models import Project
+from pootle_translationproject.models import TranslationProject
 
 
 ADMIN_URL = reverse_lazy('pootle-admin')
 
 
+def _test_admin_view(response, project):
+    request = response.wsgi_request
+    qs = TranslationProject.objects.filter(
+        project=project).order_by('pootle_path')
+    page = paginate(request, qs)
+
+    url_kwargs = {
+        'project_code': project.code,
+        'dir_path': '',
+        'filename': ''}
+
+    # bit wierd this one
```

Comment:
ill remove the comment - but the ``page`` being ``objects`` was what i was referring to mostly because the page has an ``objects_list`` var like in a django list view - so it was a bit confusing at first